src/game_constants.h

```diff
@@ -88,6 +88,18 @@
 /** Maximum range at which ranged attacks can be executed */
 #define RANGE_HARD_CAP 60
 
+/** Player view distance FLOAT */
+constexpr float player_view_distance = 60.0f;
+
+/** Player view distance INTEGER */
+constexpr int player_view_distance_int = player_view_distance;
+
+/** Distance to search for fishables from trap/fishing pole/etc */
+constexpr int fishing_distance = 60;
+
+/** Light level to use for starting a fire */
+constexpr float min_light_level_for_firestarter = 60.0f;
```

Comment:
Also move to iuse_actor.h, or move them both to iuse_constants.h, just not here.